descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,28 @@
   "id": "@artifactId@-@version@",
   "name": "Folio mod-requests-mediated module",
   "provides": [
+    {
+      "id": "request-mediated-actions",
```

Comment:
Let's rename it to requests-mediated-actions to mirror the module name - mod-requests-mediated (yaml file and controller need to be renamed too)